k8s/zero/deployment/base.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Why `StatefulSet` as opposed to `Deployment`?

Reply:
Mainly because we need PVC where we store piece of configuration that tells us how to connect to the databroker. 

I can try change to Deployment.

Reply:
after some more consideration I believe we should keep it a StatefulSet for Zero: 

1. Zero requires storage that is persistent across restarts, that keeps:
 - zero bootstrap config, so that it can know how to connect to Postgres
 - autocert data
2. The only recommended way to have a persistent storage is `StatefulSet`, i.e. see explanation [here](https://cloud.google.com/kubernetes-engine/docs/concepts/persistent-volumes#deployments_vs_statefulsets). 
3. Zero mode operation does not really require multiple replicas. we know Pomerium and Envoy can scale to very high load numbers per single instance by bumping CPU & RAM.